backend/src/lib/server-side-events.ts

```diff
@@ -276,12 +289,90 @@ export class ServerSideEvents {
       }
     }
 
+    // SORT EVENTS INTO SMALLER PACKETS
+    // SO THAT BROWSER PAGE LOADS QUICKER
+    // split events into packets
+    let parts = Object.values(this.events)
+
+    // mock a large environment
+    if (process.env.MOCK_CLUSTERS) {
+      const loaded = parts.pop()
+      parts = [...parts, ...getGiganticEvents()]
+      parts.push(loaded)
+    }
+
+    const start = parts.splice(0, 2)
+    const end = parts.pop()
```

Comment:
The hard-coded `2` appears to be here so that we send the START event and the SETTINGS event. The settings event is not always the 2nd event - if settings change, it will appear later in the event stream.

Some comments here would be good to indicate that we depend on the first and last events being certain things.